ui/src/routes/conversations/[id]/ConversationMessageInput.svelte

```diff
@@ -78,6 +78,19 @@
     text = "";
     files = [];
   }
+  function autoGrow(el) {
+      el.style.height = 'auto';
+      el.style.height = el.scrollHeight + 'px';
+
+      // Optional: clamp height to 3 lines
```

Comment:
```suggestion
      // Clamp height to maximum 5 lines
```
Doesn't seem optional as its implemented here. And looks likes its 5 lines not 3.